README.md

```diff
@@ -1,5 +1,35 @@
 
-Users can find the documentation at https://docs.ukho.dev
+We have eight software engineering teams that develop and support software using the .Net and Java stacks.  
+
+We’re proud that we’ve created this group by recruiting people who share a commitment to professional development and watched them grow with us.   Right now, we have people pursuing certification in Azure, AWS, Kubernetes and Security; we have others on management training courses; we even have people on Masters Degree programmes. 
+
+
+## How we work
+
+Everything we do is team based.  We take Agile seriously; your delivery team will be small but will have the people it needs to get things done - developers, testers, product owner and a delivery manager - as well as access to infrastructure specialists, UX experts and analysts. 
+
+Quality is paramount.  Our [engineering](./docs/software-engineering-policies/README.md) and [quality assurance](./docs/quality-assurance/README.md) policies support our teams to do good work.  Most code is done [in pairs](./docs/software-engineering-policies/PairProgramming/PairProgrammingPolicy.md) or larger groups to ensure quality and spread knowledge.  Everyone [reviews code](./docs/software-engineering-policies/CodeReview/CodeReviewPolicy.md) and everyone welcomes feedback. 
+
+We strive to automate as much as possible: [testing](./docs/quality-assurance/test-strategy.md), builds, deployments use the latest automation tools available.
+
+We actively try and pay down [technical debt](./docs/software-engineering-policies/TechnicalDebt/TechnicalDebtGuidance.md).
+
+## Our community
+
+Within this practice, we encourage the formation of other communities to support our interests and our work.
+
+The community of Lead Developers provides technical leadership for the department, recommending tools, technologies and techniques for adoption.  This group contains people with various specialisms and levels of experience who all support each other in their roles.
+
+Our [Security Champion](./docs/software-engineering-policies/SecureDevelopment/SecurityChampion.md) community is made up of people interested in [application security](./docs/software-engineering-policies/SecureDevelopment/SecureDevelopmentPolicy.md), and coaches teams to create safer software, as well as increasing their own skills and qualifications.
+
+We also have many other communities of interest covering such things as Accessibility, Testing, UX, Azure, AWS and .Net.
+
+## Balance
+
+We offer a great work / life balance.  Peoples’ start and end times are built around their own preferences and around a 10am – 2pm team time, where the team is focussed on building things (no meetings!).  We’re all set up for working from home and encourage people to make use of this if they prefer.  We are committed to providing part time options and reasonable accommodations to help people meet their needs. 
+
+
+You can find all our documentation at https://docs.ukho.dev
```

Comment:
Needed? To be reading this page they must be at that URL or the GH repo already.

Reply:
will remove